### PR TITLE
Mobile updates sidebar/account detail

### PR DESCRIPTION
**Pull request by andrewdamelio**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). ⛳️

 This link will always point to this PR's latest changes: https://peek.run/r9zhqpj

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 1]
```diff
@@ -35,23 +29,23 @@ import {
...
 
 import Styles from 'modules/app/components/app.styles.less';
 import MarketsInnerNavContainer from 'modules/app/containers/markets-inner-nav';
 import { Universe } from 'modules/types';
 
 interface AppProps {
-  blockchain: Object;
-  env: Object;
-  history: Object;
+  blockchain: object;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: We have types for these

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 2]
```diff
@@ -348,71 +327,66 @@ export default class AppView extends Component<AppProps> {
           <section className={Styles.Main}>
             <section
               className={classNames(Styles.TopBar, Styles.TopBar__floatAbove)}
-              onClick={this.mainSectionClickHandler}
-              role="presentation"
+              onClick={() => null}
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: If this click isnt used temove

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 3]
```diff
@@ -348,71 +327,66 @@ export default class AppView extends Component<AppProps> {
...
-            {universe.forkingInfo &&
-              universe.forkingInfo.forkEndTime &&
-              universe.forkingInfo.forkEndTime !== 0 &&
+            {universe.forkEndTime &&
+              universe.forkEndTime !== '0' &&
               blockchain &&
               blockchain.currentAugurTimestamp && (
-                <section className={Styles.TopBar}></section>
+                <section className={Styles.TopBar}>
+                </section>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: If no content in the section remove the closing tag

File: packages/augur-ui/src/modules/modal/add-funds.tsx [hunk 4]
```diff
@@ -7,19 +7,51 @@ import {
...
 }
 
-export const AddFunds = ({ closeAction, address }: AddFundsProps) => {
+export const AddFunds = ({
+  closeAction,
+  accountMeta,
+  address,
+}: AddFundsProps) => {
   const [selectedOption, setSelectedOption] = useState(null);
+  console.log('accountMeta', accountMeta);
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/add-funds.tsx]: Prob remove this console log

**Review by bthaile — CHANGES_REQUESTED** [review]
Lgtm, some nit pics, I’d see the correct types used. Made a comment about it.

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 5]
```diff
@@ -35,23 +29,23 @@ import {
...
+  env: object;
+  history: object;
   initAugur: Function;
   isLogged: boolean;
   isMobile: boolean;
-  location: Object;
-  loginAccount: Object;
-  modal: Object;
+  location: object;
+  loginAccount: object;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: We have a type for This

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 6]
```diff
@@ -61,14 +55,16 @@ interface AppProps {
...
-  sidebarStatus: Object;
+  sidebarStatus: object;
   updateCurrentBasePath: Function;
   updateCurrentInnerNavType: Function;
   updateMobileMenuState: Function;
   updateIsAlertVisible: Function;
   updateSidebarStatus: Function;
-  alerts: Object;
-  toasts: Array<any>;
+  alerts: object;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: There is a type for this

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 7]
```diff
@@ -35,23 +29,23 @@ import {
...
 import MarketsInnerNavContainer from 'modules/app/containers/markets-inner-nav';
-import { Universe } from 'modules/types';
+import { Universe, Blockchain } from 'modules/types';
 
 interface AppProps {
-  blockchain: Object;
-  env: Object;
-  history: Object;
+  blockchain: Blockchain;
+  env: object;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: I think we have a type for env

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 8]
```diff
@@ -35,23 +29,23 @@ import {
...
+  history: object;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: There should be a typescript type for history, but no big deal if not found

File: packages/augur-ui/src/modules/app/components/side-nav/side-nav.styles.less [hunk 9]
```diff
@@ -2,236 +2,99 @@
...
         > span {
-          display: inline-block;
-          font-weight: 500;
-          margin-left: 1rem;
-          text-align: left;
-          width: auto;
+          .text-11-bold;
+
+          align-items: center;
+          color:#848294;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/side-nav/side-nav.styles.less]: Need to use color constant

File: packages/augur-ui/src/modules/auth/components/connect-account/connect-account.tsx [hunk 10]
```diff
@@ -5,26 +5,41 @@ import Blockies from 'react-blockies';
...
 import Styles from 'modules/auth/components/connect-account/connect-account.styles.less';
 import ToggleHeightStyles from 'utils/toggle-height.styles.less';
-import { LoginAccount } from 'modules/types';
 
 interface ConnectAccountProps {
   isLogged: boolean;
   isConnectionTrayOpen: boolean;
   updateConnectionTray: Function;
+  updateMobileMenuState: Function;
+  isMobile: boolean;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/auth/components/connect-account/connect-account.tsx]: Need to use media query, dont want to keep using isMobile

File: packages/augur-ui/src/modules/auth/containers/connect-account.ts [hunk 11]
```diff
@@ -4,14 +4,18 @@ import {
   updateAuthStatus,
   IS_CONNECTION_TRAY_OPEN,
 } from 'modules/auth/actions/auth-status';
+import { updateMobileMenuState } from 'modules/app/actions/update-sidebar-status';
 
 const mapStateToProps = state => ({
   isLogged: state.authStatus.isLogged,
   userInfo: state.loginAccount.meta,
   isConnectionTrayOpen: state.authStatus.isConnectionTrayOpen,
+  isMobile: state.appStatus.isMobile,
```
Comment (bthaile) [inline packages/augur-ui/src/modules/auth/containers/connect-account.ts]: Lets get away from isMobile. I think we are moving toward Media queries. @phoebemirman  thoughts?

**Review by bthaile — CHANGES_REQUESTED** [review]
Few more types to work out. Not a fan of isMobile its expensive to maintain that. Basically a constant monitoring of resize hook. If mem serves. I’d like to get @phoebemirman  thoughts on the usage of isMobile in this PR

File: packages/augur-ui/src/modules/auth/components/connect-account/connect-account.tsx (hunk 10 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/modules/auth/components/connect-account/connect-account.tsx]: refactored away 👍

File: packages/augur-ui/src/modules/auth/containers/connect-account.ts (hunk 11 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/modules/auth/containers/connect-account.ts]: agree

Comment (andrewdamelio) [reply packages/augur-ui/src/modules/auth/containers/connect-account.ts]: removed

**Review by bthaile — APPROVED** [review]
Lgtm, nice 👍

**Pull request merged**